clients/http-support/src/main/java/org/eclipse/sw360/http/RequestBodyBuilderImpl.java

```diff
@@ -53,6 +55,7 @@
      */
     public RequestBodyBuilderImpl(ObjectMapper mapper) {
         this.mapper = mapper;
+        mapper.setPropertyNamingStrategy(PropertyNamingStrategy.LOWER_CAMEL_CASE);
```

Comment:
I am not sure, but it looks like this will be applicable for all not only components. Need to make sure it does not affect others.

Reply:
Thanks for the crosscheck and confirmation.